pytype/tools/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ py_library(
     .environment
     .runner
     .tool_utils
+    .path_tools
```

Comment:
the `tools` directory is for standalone tools that build on top of pytype. pytype itself should not depend on anything in this subtree. also for naming consistency, call the module `path_utils` rather than `path_tools`

if you would like to group the win32 wrappers together you could create a new `pytype/platform_utils` directory, otherwise just move both these modules to the top level `pytype/`